<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="IT-SANTOSHB" name="in.dishtv.Activation.Installation_add_Installation_Call_Test" tests="3" failures="1" timestamp="15 May 2019 11:05:25 GMT" time="9.677" errors="0">
  <testcase name="installation_GetTittle" time="0.134" classname="in.dishtv.Activation.Installation_add_Installation_Call_Test">
    <failure type="java.lang.AssertionError" message="We are not in Dealer Page:::Title Mistatch::: expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: We are not in Dealer Page:::Title Mistatch::: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at in.dishtv.Activation.Installation_add_Installation_Call_Page.check_title(Installation_add_Installation_Call_Page.java:232)
	at in.dishtv.Activation.Installation_add_Installation_Call_Test.installation_GetTittle(Installation_add_Installation_Call_Test.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- installation_GetTittle -->
  <testcase name="installation_individual_multi" time="9.114" classname="in.dishtv.Activation.Installation_add_Installation_Call_Test"/>
  <testcase name="Installation_mousehover" time="0.429" classname="in.dishtv.Activation.Installation_add_Installation_Call_Test"/>
</testsuite> <!-- in.dishtv.Activation.Installation_add_Installation_Call_Test -->
